src/pages/MyEarnings/TotalEarningsAndChainSelect.tsx

```diff
@@ -0,0 +1,182 @@
+import { t } from '@lingui/macro'
+import { rgba } from 'polished'
+import { useCallback, useEffect, useRef } from 'react'
+import { useDispatch } from 'react-redux'
+import { Button, Flex } from 'rebass'
+import earningApi, { useLazyGetElasticEarningQuery, useLazyGetElasticLegacyEarningQuery } from 'services/earning'
+import styled from 'styled-components'
+
+import { ReactComponent as RefreshIcon } from 'assets/svg/refresh.svg'
+import { MouseoverTooltip } from 'components/Tooltip'
+import { useActiveWeb3React } from 'hooks'
+import useMixpanel, { MIXPANEL_TYPE } from 'hooks/useMixpanel'
+import useTheme from 'hooks/useTheme'
+import ShareTotalEarningsButton from 'pages/MyEarnings/ShareTotalEarningsButton'
+import { useAppSelector } from 'state/hooks'
+
+// TODO: move to common
+const formatPercent = (value: number) => {
+  const formatter = Intl.NumberFormat('en-US', {
+    notation: 'standard',
+    style: 'percent',
+    minimumFractionDigits: 0,
+    maximumFractionDigits: 2,
+  })
+
+  return formatter.format(value)
+}
+
+const formatValue = (value: number) => {
+  const formatter = Intl.NumberFormat('en-US', {
+    notation: 'standard',
+    style: 'currency',
+    currency: 'USD',
+    minimumFractionDigits: 0,
+    maximumFractionDigits: 8,
+  })
+
+  return formatter.format(value)
+}
+
+const Value = styled.span`
+  max-width: 100%;
+  display: inline-flex;
+  font-weight: 500;
+  font-size: 36px;
+  line-height: 44px;
+  flex: 0 0 max-content;
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
+`
+
+const RefreshButton = () => {
+  const theme = useTheme()
+  const { mixpanelHandler } = useMixpanel()
+  const dispatch = useDispatch()
+  const { account } = useActiveWeb3React()
+  const selectedChainIds = useAppSelector(state => state.myEarnings.selectedChains)
+  const [elasticTrigger, elasticData] = useLazyGetElasticEarningQuery()
+  const [elasticLegacyTrigger, elasticLegacyData] = useLazyGetElasticLegacyEarningQuery()
+  // const [classicTrigger, classicData] = useLazyGetClassicEarningQuery()
```

Comment:
```suggestion
```

Reply:
Intentionally keep this so that we won't miss